var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -76,6 +76,43 @@ class Trilinos(CMakePackage):
     variant('xsdkflags',    default=False,
             description='Compile using the default xSDK configuration')
 
+    # CUDA options (compare to kokkos/package.py)
+    variant('cuda', default=False, description="enable Cuda backend")
+    # kokkos has conflicts with cuda for kokkos@2.5:2.7, restrict this variant
+    # to @12.14.1 or later.
+    conflicts('+cuda', when='@:12.14.0')
+    conflicts('+cuda', when='~kokkos')
+    variant('kokkos_enable_cuda_uvm', default=False,
+            description="set force_uvm Kokkos CUDA option")
+    variant('kokkos_enable_rdc', default=False,
+            description="set rdc Kokkos CUDA option")
+    variant('kokkos_enable_cuda_lambda', default=False,
```

Comment:
As Tpetra uses `KOKKOS_LAMBDA`, shouldn't this be `True` by default?

Reply:
See my previous comment.